api/specs.proto

```diff
@@ -175,6 +175,9 @@ message ContainerSpec {
 
 	// PullOptions parameterize the behavior of image pulls.
 	PullOptions pull_options = 10;
+
+	// Privileged give extended privileges to the container.
+	bool Privileged = 11;
```

Comment:
Field names in protobuf should be lowercase (`privileged`). They automatically get converted to go-style CamelCase names when the Go code is generated.